sdk/monitor/azure-monitor-query/azure/monitor/query/aio/_logs_query_client_async.py

```diff
@@ -53,9 +53,9 @@ async def query_workspace(
         workspace_id: str,
         query: str,
         *,
-        timespan: Union[
+        timespan: Optional[Union[
```

Comment:
Here, since no default is provided, `timespan` is a required keyword argument that can explicitly be None (indicating the query shouldn't be scoped to any specific time range). Looks like most of the query samples explicitly use `timespan=None` and at some point in the past, defaulting `timespan` to `None` was removed.

To accommodate this, I was trying to follow the guidance here: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/static_type_checking.md#use-typingoptional-when-a-parameter-can-explicitly-be-none

I will update the docstrings to mention the usage of `None` as a possible argument.

Perhaps adding `None` to the Union would be preferred instead of using `Optional`?

Reply:
Ah I didn't realize `None` was a valid value - in that case this is fine :)